src/youtube/players/shared/youtubeTrackingEvents.ts

```diff
@@ -1,5 +1,5 @@
 import { track } from '@fandom/tracking-metrics/tracking/dataLayer';
-import { YoutubeTakeOverDetails } from 'loaders/utils/GetYoutubeTakeoverDetails';
+import { YoutubeTakeOverDetails } from 'youtube/types';
```

Comment:
@p-forfa It removes problems while building in the console. Am I right thinking it shouldn't break anything? How can I double-check it? ;-)

Reply:
Ahh, good catch. I'm not sure why that wasn't flagged by my IDE. By running 'yarn build', it should be enough since this is a typescript definition.

The other good way to actually check on the player itself, is to import the the 'DesktopArticleVideoLoader' component inside /test-jw/src/App.js, and run 'yarn app' from the source directory. 

It should open up a localhost page. Add this query param `?youtube_embed_video_id=F0Ga_nPZuiI` where `F0Ga_nPZuiI` is some youtube video id. This will have the Youtube Video Embed takeover. 

I'll document the local Fandom JWPlayer process in the README so its clearer.

Reply:
Seems working 👍